dali/test/dali_test_config.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <mutex>
+#include <cstring>
+#include <cassert>
+#include "dali/test/dali_test_config.h"
+
+namespace dali {
+namespace testing {
+namespace program_options {
+
+namespace {
+
+/// Path to Dali_extra repository
+std::string _dali_extra_path;  // NOLINT
+
+std::once_flag parse_only_once;
+
+bool parsed;  /// indicates, that parsing completed successfully
+
+
+void parse(int argc, const char **argv) {
+  // TODO(mszolucha): in case more args appear, use better solution (e.g. boost::program_options)
+  const size_t cmd_len = 17;
+  for (int i = 1; i < argc; i++) {
+    if (0 == std::strncmp(argv[i], "--dali_extra_path", cmd_len)) {
+      _dali_extra_path = std::string{&argv[i][cmd_len + 1]};
+      parsed = true;
+      break;
+    }
+  }
+}
+
+}  // namespace
+
+
+const std::string &dali_extra_path() {
+  assert(parsed);
```

Comment:
I think we don't need that. If there's no parameter, we can assume working directory is the base dir.

Reply:
Removed